### PR TITLE
Fix antibot crash after map change

### PR DESCRIPTION
**Pull request by heinrich5991**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1151,6 +1151,14 @@ target_link_libraries(${TARGET_SERVER} ${LIBS_SERVER})
 list(APPEND TARGETS_OWN ${TARGET_SERVER})
 list(APPEND TARGETS_LINK ${TARGET_SERVER})
 
+if(ANTIBOT)
+  if(CMAKE_VERSION VERSION_LESS 3.14 AND (CMAKE_VERSION VERSION_GREATER 3.8 OR CMAKE_VERSION VERSION_EQUAL 3.8))
+    set_property(TARGET ${TARGET_SERVER} PROPERTY BUILD_RPATH "$ORIGIN")
+  else()
+    message(STATUS "Can't set BUILD_RPATH in CMake before 3.8, pass -Wl,-rpath,'$ORIGIN' manually if you wish to emulate this. Or just install a newer version of CMake...")
```
Comment (def-) [inline CMakeLists.txt]: Can't you pass -Wl-,-rpath,'$ORIGIN' to https://cmake.org/cmake/help/v3.10/variable/CMAKE_EXE_LINKER_FLAGS.html#variable:CMAKE_EXE_LINKER_FLAGS and https://cmake.org/cmake/help/v3.10/variable/CMAKE_SHARED_LINKER_FLAGS.html#variable:CMAKE_SHARED_LINKER_FLAGS instead?

File: CMakeLists.txt [hunk 2]
```diff
@@ -1151,6 +1151,14 @@ target_link_libraries(${TARGET_SERVER} ${LIBS_SERVER})
 list(APPEND TARGETS_OWN ${TARGET_SERVER})
 list(APPEND TARGETS_LINK ${TARGET_SERVER})
 
+if(ANTIBOT)
+  if(CMAKE_VERSION VERSION_LESS 3.14 AND (CMAKE_VERSION VERSION_GREATER 3.8 OR CMAKE_VERSION VERSION_EQUAL 3.8))
```
Comment (Learath2) [inline CMakeLists.txt]: Why is `v < 3.14` needed too?

File: CMakeLists.txt (hunk 1 above)
Comment (heinrich5991) [reply CMakeLists.txt]: @def- Then it becomes the default for installed executables as well. `BUILD_RPATH` only applies to binaries before calling `make install`.

File: CMakeLists.txt (hunk 2 above)
Comment (heinrich5991) [reply CMakeLists.txt]: For `>= 3.14` we have the proper solution in CMake, see below: https://github.com/ddnet/ddnet/pull/2084/commits/bda3adabccc373d3f097c18c727975c77819d225#diff-af3b638bc2a3e6c650974192a53c7291R1690-R1692.

**Pull request merged**